sound/soc/sdw_utils/Kconfig

```diff
@@ -0,0 +1,6 @@
+# SPDX-License-Identifier: GPL-2.0-only
+config SND_SOC_SDW_UTILS
+	tristate "SoundWire ASoC common helper functions"
```

Comment:
just use 'tristate', this has to be selected by other drivers and cannot be exposed as an option.

Reply:
will change it.